extension.js

```diff
@@ -730,19 +730,21 @@ const EmojisMenu = new Lang.Class({
 		this.emojiCategories = [];
 	
 		/* creating new categories with emojis loaded in EMOJIS_CHARACTERS */
-		this.emojiCategories[0] = new EmojiCategory(	_('Smileys & Body'),		'face-smile-symbolic',			0	);
-		this.emojiCategories[1] = new EmojiCategory(	_('Peoples & Clothing'),	'contact-new-symbolic',			1	);	
-		this.emojiCategories[2] = new EmojiCategory(	_('Animals & Nature'),		'face-monkey-symbolic',			2	);	
-		this.emojiCategories[3] = new EmojiCategory(	_('Food & Drink'), 			'my-caffeine-on-symbolic',		3	);
-		this.emojiCategories[4] = new EmojiCategory(	_('Travel & Places'), 		'airplane-mode-symbolic',		4	);
-		this.emojiCategories[5] = new EmojiCategory(	_('Activities & Sports'),	'applications-games-symbolic',	5	);
-		this.emojiCategories[6] = new EmojiCategory(	_('Objects'),				'alarm-symbolic',				6	);
-		this.emojiCategories[7] = new EmojiCategory(	_('Symbols'),				'emblem-default-symbolic',		7	);
-		this.emojiCategories[8] = new EmojiCategory(	_('Flags'),					'flag-symbolic',				8	);
+		this.emojiCategories[0] = new EmojiCategory(	_('Smileys'),				'face-smile-symbolic',							0);
+		this.emojiCategories[1] = new EmojiCategory(	_('Body'),		   			'preferences-system-privacy-symbolic',			1);
+		this.emojiCategories[2] = new EmojiCategory(	_('People'),				'avatar-default-symbolic',						2);
+		this.emojiCategories[3] = new EmojiCategory(	_('Animals'),				'face-monkey-symbolic',							3);
+		this.emojiCategories[4] = new EmojiCategory(	_('Nature'),				'preferences-desktop-screensaver-symbolic',		4);
+		this.emojiCategories[5] = new EmojiCategory(	_('Food & Drink'), 			'my-caffeine-on-symbolic',						5);
+		this.emojiCategories[6] = new EmojiCategory(	_('Travel & Places'), 		'airplane-mode-symbolic',						6);
+		this.emojiCategories[7] = new EmojiCategory(	_('Activities & Sports'),	'emoji-activities-symbolic',					7);
+		this.emojiCategories[8] = new EmojiCategory(	_('Objects'),				'alarm-symbolic',								8);
+		this.emojiCategories[9] = new EmojiCategory(	_('Symbols'),				'emblem-default-symbolic',						9);
+		this.emojiCategories[10] = new EmojiCategory(	_('Flags'),					'flag-symbolic',								10);
 	},
 	
 	_addAllCategories: function() {
-		for (let i = 0; i< 9; i++) {			
+		for (let i = 0; i< 11; i++) {			
```

Comment:
I believe `< this.emojiCategories.length` (or even a `forEach`, for this case) would be more appropriate here (so in the future people don't need to worry about modifying this. The same goes for line 758 below.
I'm new to the project though, I don't know if there is any limitation to do this.